tox.ini

```diff
@@ -51,22 +51,6 @@ basepython = python
 passenv = *
 whitelist_externals = bash
 deps =
-    -rrequirements-tests.txt
+    -rrequirements/tests.txt
     djongo
 commands = {posargs:bash -x functional.sh}
-
-[testenv:build]
-basepython = python
-skip_install = true
-deps =
-    build
-commands =
-    {envpython} -m build
```

Comment:
Any reason not to have these commands in `tox` still (so that they can be run locally just as easily) and then within GitHub Actions run `tox -e build`, `tox -e upload`. In general I think `tox` is a nice place to put scripts like this as it is discoverable and also easy to use/test from a local computer.

But I don't have a strong preference here if this is what you prefer.

Reply:
We can't really use the upload command within Jazzband, due to the PyPI password being unavailable to us.

And then build command is a one-liner so a shortcut for that doesn't make sense in my opinion.

Reply:
That is a good point. 👍